Package.swift

```diff
@@ -5,6 +5,9 @@ import PackageDescription
 
 let package = Package(
     name: "swift-cubic",
+    platforms: [
+        .iOS(.v10),
```

Comment:
Based on [this release by Apple](https://developer.apple.com/news/?id=03202019a), shouldn't the minimum version be at least 12.1? Or am I misinterpreting this version number?

Reply:
On second thought, I am also questioning why grpc/swift-cubic needs to have a dependency on iOS. That seems like it will prohibit a developer from creating a Mac OS app from the SDK.

Reply:
This Apple's release is only about new released apps. A developer, who uses the SDK, should take care about this. But if he already has an app in the App Store, he is able to import our SDK and release an update without increasing the minimum iOS version.

Reply:
I will add MacOS version